sdk/webpubsub/azure-messaging-webpubsub/swagger/webpubsub.json

```diff
@@ -0,0 +1,717 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Web PubSub Service REST API",
+    "version": "2020-10-01"
+  },
+  "paths": {
+    "/api/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_GetHealthStatus",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "api-version",
+            "type": "string",
+            "default": "2020-10-01"
```

Comment:
do we have a new preview api-version for it?

Reply:
this swagger is now removed and the README now points to the api-specs swagger file.